modules/installation-configuration-parameters.adoc

```diff
@@ -1159,6 +1203,31 @@ The GCP Compute Engine System service account email, like `<service_account_name
 |For compute machines, the ID of the project in which the KMS key ring exists. This value defaults to the VM project ID if not set.
 |The GCP project ID.
 
+|`compute.platform.gcp.osDisk.encryptionKey.kmsKeyServiceAccount`
+|The GCP Compute Engine System service account used for the encryption request for compute machines. For more information on GCP service accounts, see Google's documentation on link:https://cloud.google.com/compute/docs/access/service-accounts#compute_engine_service_account[service accounts].
```

Comment:
Suggest to replace `The GCP Compute Engine System service account used for the encryption request for compute machines.` with `KMSKeyServiceAccount is the service account being used for the encryption request for the given KMS key. If absent, the Compute Engine default service account is used.`.

Reply:
Same change as above.